client/index.js

```diff
@@ -0,0 +1,6 @@
+/* eslint-disable */
```

Comment:
Why are you having this here?

Reply:
That was the file I created so that we could test our notification on real-time